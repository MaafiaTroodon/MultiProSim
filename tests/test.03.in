2 5 2
Proc1 2 1 1
SEND 201
HALT

Proc2 2 1 2
RECV 101
HALT
