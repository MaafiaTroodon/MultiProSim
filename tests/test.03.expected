[01] 00000: process 1 new
[01] 00000: process 1 ready
[01] 00000: process 1 running
[01] 00001: process 1 blocked (send)
[01] 00002: process 1 finished
[02] 00000: process 1 new
[02] 00000: process 1 ready
[02] 00000: process 1 running
[02] 00001: process 1 blocked (recv)
[02] 00002: process 1 finished
| 00002 | Proc 01.01 | Run 1, Block 0, Wait 0, Sends 1, Recvs 0
| 00002 | Proc 02.01 | Run 1, Block 0, Wait 0, Sends 0, Recvs 1
